.nvmrc

```diff
@@ -1 +1 @@
-8.9.1
+8.10.0
```

Comment:
Should we use 8.15.0?

Reply:
This was the smallest bump. We could probably go higher if we wanted, but 🤷‍♂️